script/server

```diff
@@ -38,6 +38,7 @@
 
 main()
 {
+    # shellcheck source=/dev/null
```

Comment:
Just curious - did something change to require these exclusions?

Reply:
In a previous push, `make-pretty` complained about this (even in some unchanged files) so I made such changes. I don't know what will happen if I revert these `# shellcheck` changes.

https://github.com/openthread/ot-br-posix/runs/4862547983?check_suite_focus=true